SimGeneral/PreMixingModule/test/BuildFile.xml

```diff
@@ -0,0 +1,13 @@
+<library file="TestPreMixingPileupAnalyzer.cc" name="SimGeneralPreMixingModuleTestPlugins">
+  <flags EDM_PLUGIN="1"/>
+  <use name="DataFormats/Common"/>
+  <use name="FWCore/Framework"/>
+  <use name="FWCore/ParameterSet"/>
+  <use name="FWCore/Utilities"/>
+  <use nam="SimDataFormats/PileupSummaryInfo"/>
```

Comment:
@makortel , is it typo or not? "use nam="

Reply:
Thanks for catching, fixed.